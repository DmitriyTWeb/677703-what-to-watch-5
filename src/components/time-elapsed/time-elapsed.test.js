import React from "react";
import renderer from "react-test-renderer";
import TimeElapsed from "./time-elapsed";

it(`Should TimeElapsed render correctly`, () => {
  const tree = renderer.create(
      <TimeElapsed
        currentTime={``}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
