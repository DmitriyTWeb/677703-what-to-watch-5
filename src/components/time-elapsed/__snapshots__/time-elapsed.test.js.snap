// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should TimeElapsed render correctly 1`] = `
<div
  className="player__time-value"
>
  00:00:00
</div>
`;
